cves/CVE-2010-3411.yml

```
@@ -197,4 +214,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: The mistakes that were made that led to this vulnerability were coding
```

Formatting: Multi-line text entries should put to the next line using the | for readability.
```suggestion
  answer: |
      The mistakes that were made that led to this vulnerability were coding
```